modules/options/base.go

```diff
@@ -79,17 +81,53 @@ func walkAssetDir(root string, callback func(path, name string, d fs.DirEntry, e
 	return nil
 }
 
-func statDirIfExist(dir string) ([]string, error) {
-	isDir, err := util.IsDir(dir)
+// mustLocalPathAbs coverts a path to absolute path
+// FIXME: the old behavior (StaticRootPath might not be absolute), not ideal, just keep the same as before
+func mustLocalPathAbs(s string) string {
+	abs, err := filepath.Abs(s)
 	if err != nil {
-		return nil, fmt.Errorf("unable to check if static directory %s is a directory. %w", dir, err)
+		log.Fatal("Unable to get absolute path for %q: %v", s, err)
```

Comment:
:eyes: Interesting. I wonder how many users will suddenly report that their instance doesn't start anymore…

Reply:
I do not think so. At the moment, only "StaticRootPath" is affected with "dynamic.go".

Most users won't be affected.

Even if some one is affected, the "relative path" behavior is already very dangerous, this panic will help them to set their StaticRootPath to an absolute one.

Reply:
To be clear, almost 100% users won't be affected. Because, `filepath.Abs` would never fail in a real system.

If it fails, the user must already be in trouble.